hack/windows/Prepare-Node.ps1

```diff
@@ -157,6 +157,10 @@ Write-Host "Registering kubelet service"
 nssm install kubelet $global:Powershell $global:PowershellArgs $global:StartKubeletScript
 
 nssm set kubelet DependOnService $ContainerRuntime
+nssm set kubelet AppStdout C:\var\log\kubelet\kubelet.log
+nssm set kubelet AppStdoutCreationDisposition 2
```

Comment:
could you add a comment there explaining what this does (setting `AppStdoutCreationDisposition` to 2)?

Reply:
added

Reply:
@antoninbas could you share your thoughts on [this ask](https://github.com/antrea-io/antrea/issues/6219#issuecomment-2055739751) about if we need to keep collecting kubelet logs on Windows in the support bundle?